doc/user_guide/user_guide.md

```diff
@@ -1,3 +1,239 @@
+2022-04-27 18:07:09 (067.368ms) [INFO]   Reading EXA_METADATA
+2022-04-27 18:07:09 (078.316ms) [INFO]     - row 1: maxBinaryLiteralLength = '0'
+2022-04-27 18:07:09 (078.344ms) [INFO]     - row 2: maxCatalogNameLength = '128'
+2022-04-27 18:07:09 (078.352ms) [INFO]     - row 3: maxCharLiteralLength = '2000000'
+2022-04-27 18:07:09 (078.359ms) [INFO]     - row 4: maxColumnNameLength = '128'
+2022-04-27 18:07:09 (078.374ms) [INFO]     - row 5: maxColumnsInGroupBy = '0'
+2022-04-27 18:07:09 (078.499ms) [INFO]   Connection closed successfully
+2022-04-27 18:07:09 (078.513ms) [INFO]   Environment closed successfully
+```
+
+## Usage
+
+`exasol-driver-lua` follows the API of [LuaSQL](https://keplerproject.github.io/luasql/), so the [LuaSQL manual](https://keplerproject.github.io/luasql/manual.html) is a good starting point.
+
+The following describes the basic usage. See the API documentation for a detailed description of each method. (Will be added in [#46](https://github.com/exasol/exasol-driver-lua/issues/46))
+
+### Configure Logging
+
+The driver uses [remotelog](https://github.com/exasol/remotelog-lua#readme) for logging, so you can configure the log level like this:
+
+```lua
+local log = require("remotelog")
+log.set_level("INFO")
+```
+
+See remotelog's [user guide](https://github.com/exasol/remotelog-lua/blob/main/doc/user_guide/user_guide.md) for details.
+
+### Creating an Environment
+
+The environment allows you to create database connections:
+
+```lua
+-- Import the library
+local driver = require("luasqlexasol")
+-- Create a new environment
+local environment = driver.exasol()
+
+-- Use the environment to create database connections...
+```
+
+### Creating a Connection
+
+You can create a database connection using the environment:
+
+```lua
+-- Create a new connection with default properties
+local connection, err = environment:connect("<hostname>:8563", "<username>", "<password>")
+
+-- Create a new connection with custom properties
+local properties = {tls_verify = "none", tls_protocol = "tlsv1_2", tls_options = "no_tlsv1"}
+local connection, err = environment:connect("<hostname>:8563", "<username>", "<password>", properties)
+
+if err ~= nil then
+  -- Handle connection error
+end
+```
+
+The `connect()` method expects four arguments:
+
+1. The source name consists of hostname or IP address of the database and the port.
+2. The name of the database user.
+3. The password of the database user.
+4. An optional table with connection properties, see below for details.
+
+The `connect()` returns two results:
+
+1. The connection object if the connection was established successfully or `nil` in case of an error
+2. An error in case the operation failed.
+
+#### Connection Properties
+
+When creating a new connection you can specify the following properties:
+
+* `tls_verify` specifies how the database's TLS certificate should be verified. Possible values are:
+  * `none` (default)
```

Comment:
is this a good default?

Reply:
No. I created https://github.com/exasol/exasol-driver-lua/issues/53 to investigate this.